package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "did-resolver",
-  "version": "0.0.6",
+  "version": "0.1.0-alpha-1",
```

Comment:
Change to `0.1.0-alpha.0`
Alpha number should be separated with a `.` instead of a `-` and start from 0.